frontend/src/components/Navbar/index.scss

```
@@ -0,0 +1,50 @@
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 1291px;
```

szerokość navbara raczej nie może być w pixelach, gdyż na niektórych monitorach będzie wychodziło poza ekran, a na innych nie będzie wypozycjonowany maksymalnie do prawej, sugeruję coś takiego:
```suggestion
  right: 0;
  top: 0;
  left: 0;
```